components/camel-zipfile/src/test/java/org/apache/camel/dataformat/zipfile/ZipFileDataFormatTest.java

```diff
@@ -187,6 +217,35 @@ public void setUp() throws Exception {
         deleteDirectory(TEST_DIR);
         super.setUp();
     }
+    
```

Comment:
try-with-resources will simplify this code:

`
    private static void copy(File file, OutputStream out) throws IOException {
        try (InputStream in = new FileInputStream(file)) {
            copy(in, out);
        }
    }
    private static void copy(InputStream in, File file) throws IOException {
        try (OutputStream out = new FileOutputStream(file)) {
            copy(in, out);
        }
    }
`